internal/bminventory/inventory_test.go

```diff
@@ -4772,41 +4778,12 @@ var _ = Describe("GetClusterInstallConfig", func() {
 		ctrl.Finish()
 	})
 
-	It("returns the correct install config", func() {
+	It("check get install config flow", func() {
 		params := installer.GetClusterInstallConfigParams{ClusterID: clusterID}
+		mockInstallConfigBuilder.EXPECT().GetInstallConfig(gomock.Any(), false, "").Return([]byte("some string"), nil).Times(1)
 		response := bm.GetClusterInstallConfig(ctx, params)
-		actual, ok := response.(*installer.GetClusterInstallConfigOK)
-		Expect(ok).To(BeTrue())
-
-		config := installcfg.InstallerConfigBaremetal{}
```

Comment:
You seem to have removed quite a few tests. Is this functionality covered elsewhere? E.g. default config. Maybe the test should be moved to installcfg_test.go?

Reply:
since there was no interface in the installcfg, the unit tests in the bminventory also checked installcfg functionality. in addition, there are unit tests in the installcfg that checked those kinds of input, so now we can remove them from the bminventory